### PR TITLE
Add link to companies list and fix misspellings

### PR DESCRIPTION
**Pull request by maximelafarie**
Because it's so much better to have a clickable picture! It also avoids to confound different companies through their logos (e.g. BiiG and Saveur-Bière) and makes them more visible. 🖖

### CODE REVIEW

File: src/components/home/References.js [hunk 1]
```diff
@@ -8,10 +8,10 @@ const openSourceList = openSourceData.map(({ name, logo, link }) => (
   </a>
 ));
 
-const otherList = otherData.map(({ name, logo }) => (
-  <div key={name} className="references__item">
+const otherList = otherData.map(({ name, logo, link }) => (
+  <a href={link} target="_blank" key={name} className="references__item">
```
Comment (gaetanmarsault) [inline src/components/home/References.js]: <a href={link} target="_blank" title="{name} (new window)" key={name} className="references__item">

File: src/components/home/References.js [hunk 2]
```diff
@@ -8,10 +8,10 @@ const openSourceList = openSourceData.map(({ name, logo, link }) => (
   </a>
 ));
 
-const otherList = otherData.map(({ name, logo }) => (
-  <div key={name} className="references__item">
+const otherList = otherData.map(({ name, logo, link }) => (
+  <a href={link} target="_blank" title={`${name} (new window)`} key={name} className="references__item">
```
[the remark this answers is not in the document]
Comment (maximelafarie) [reply src/components/home/References.js]: Sure!

[the remark this answers is not in the document]
Comment (maximelafarie) [reply src/components/home/References.js]: Done! 👌

**Comment by dunglas** [comment]
Thanks @maximelafarie

**Pull request merged**